CHANGELOG.rst

```diff
@@ -16,6 +16,13 @@ Subsections for each version can be one of the following;
 
 Each individual change should have a link to the pull request after the description of the change.
 
+1.2.2 (2024-02-15)
+------------------
+Changed
+^^^^^^^
+- Changed LogTransformer to use log1p(x) instead of log(x+1) `#178 <https://github.com/lvgig/tubular/pull/178>`_
+- Changed unit tests using log(x+1) to log1p(x) `#178 <https://github.com/lvgig/tubular/pull/178>`_
+
```

Comment:
Also looks like the last release went out without updating the changelog.  Could you update for 1.2.1 too please?

Reply:
looking at the merge conflicts there appears to have been the 1.2.1 changelog added at some point